### PR TITLE
Added split method

### LINKED ISSUE #120: Adding Split method in string
I want to add new method of string, which is split() method
@sumn2u can you assign me?

### PR DESCRIPTION
**Pull request by Chandanmali**
fixed #120 
Added split method in string please review it.

### CODE REVIEW

File: guess.CM_NUMBER/index
Comment (sumn2u) [inline guess.CM_NUMBER/index:1]: Can you help me to understand what this is for?

**Review by sumn2u — COMMENTED** [review]
I see there's an addition of `guess.CM_NUMBER`. Can you help me to understand this?

**Comment by sumn2u** [comment]
Is it also possible to delete the whole `guess.CM_NUMBER` folder? If you are planning to show an example project, i suggest to use embed via `codepen` or `codesandbox`.

**Comment by Chandanmali** [comment]
only add split method in string

**Review by sumn2u — APPROVED** [review]
LGTM

**Pull request merged**